mbed_host_tests/host_tests_runner/mbed_base.py

```diff
@@ -60,6 +60,8 @@ def __init__(self, options):
             self.serial_baud = int(port_config[1])
             self.serial_timeout = float(port_config[2])
 
+        self.serial_baud = self.options.baud_rate if self.options.baud_rate else self.serial_baud
```

Comment:
Please put a comment here saying "Override baud rate with user supplied value" or something like that.